test/powershell/Modules/Microsoft.PowerShell.Utility/assets/TestImportCsv1.csv

```diff
@@ -0,0 +1,4 @@
+﻿data1,1,A
```

Comment:
Descriptive file names would be nice to have here. Something like:
TestImportCsv1 --> TestImportCsv_NoHeader
TestImportCsv2 --> TestImportCsv_WithHeader
TestImportCsv3 --> TestImportCsv_W3C_ELF